.github/CONTRIBUTING.md

```diff
@@ -1,388 +1,98 @@
 # Contributing
 
-## Requirements
+## Prerequisites
 
-Set up your SSH Key GitHub account and install node.js 4 or higher.
+To contribute to Carbon, you'll need to install:
 
-- [Generating SSH Keys - GitHub](https://help.github.com/articles/generating-ssh-keys/)
-- [`nvm` (Node Version Manager)](https://github.com/creationix/nvm) to use the
-  `Node 6`.
+- [node(v10 or higher)](https://nodejs.org/en/download/)(you can download it
+  here or follow their installation through a package manager
+  [here](https://nodejs.org/en/download/package-manager/))
+  - We recommend using
+    [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md) as your version
+    manager for Node.
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [yarn](https://yarnpkg.com/en/docs/install#mac-stable)
 
-Contributing to carbon-components requires that you can run this repo locally on
-your computer.
+You will also need to have a GitHub account. Head over to
+[github.com](https://github.com/) to make an account.
 
-## Coding style
+Finally you'll need a code editor to make changes to Carbon. There are many to
```

Comment:
Should we highlight editor stuff? Same question as above, unsure what level we expect folks to be contributing from.

Reply:
I wasn't sure how beginner friendly we wanted to make this, I can remove this if you think it's too basic!

Reply:
I would like to keep the section about code editors. In my programming classes, it was the thing people had no idea about (and the class never talked about it) so i think another place that mentions what to use is better. Plus it's pretty easy to ignore if it doesn't apply to them